packages/react-native/src/notifier.js

```diff
@@ -97,32 +98,45 @@ const Bugsnag = {
       Bugsnag._client._logger.warn('Bugsnag.start() was called more than once. Ignoring.')
       return Bugsnag._client
     }
+
     if (!isDebuggingRemotely) {
       Bugsnag._client = createClient(opts)
       return Bugsnag._client
-    } else {
-      console.warn(REMOTE_DEBUGGING_WARNING)
-      let initialised = false
-      const stubClient = {}
-      CLIENT_METHODS.reduce((accum, m) => {
-        stubClient[m] = new Proxy(() => {
-          console.warn(
-            `This call to Bugsnag.${m}() is a no-op because remote debugging is enabled and Bugsnag has not yet initialized`
-          )
-        }, {
-          apply: (target, thisArg, argumentsList) => {
-            if (!initialised) return Reflect.apply(target, thisArg, argumentsList)
-            return Reflect.apply(Bugsnag._client[m], Bugsnag._client, argumentsList)
+    }
+
+    console.warn(REMOTE_DEBUGGING_WARNING)
+
+    let initialised = false
+
+    const stubSchema = { ...schema }
+    delete stubSchema.apiKey
```

Comment:
It's not obvious to me why we need to remove the apiKey here; this could use a comment

Reply:
Comment added.